stock_release_channel_process_end_date/models/stock_release_channel.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class StockReleaseChannel(models.Model):
+
+    _inherit = "stock.release.channel"
+
+    process_end_date = fields.Datetime(
```

Comment:
@rousseldenis A release channel instance is not created every day. A release channel is a way to organize your pick to deliver process into your company. It's therefore useless to specify the date and time when the process of pickings should be completed. IMO, this process_end_date should be replaced by a process_end_time... 
@jbaudoux What do you thing about?

Reply:
> @rousseldenis A release channel instance is not created every day. A release channel is a way to organize your pick to deliver process into your company. It's therefore useless to specify the date and time when the process of pickings should be completed. IMO, this process_end_date should be replaced by a process_end_time...

@lmignon Ok, didn't catch that.

The only drawback I see is that is not flexible (but maybe it's wanted - I don't have the whole flow in mind) as one may want to have channel opened during the day, but pickings done during the night (so, overlap between two days)